custom_image/storage/README.md

```diff
@@ -0,0 +1,103 @@
+   2. For **Installed Version**, select the version for your custom image creation.
+   3. For **Platform**, select Linux 64-bit,x86_64.
+   4. Click **Continue**, and it will redirect to _Select fixes_ page.
+   5. On the _Select fixes_ page, click the **Data Management** link to get the fix pack.
+
+5. Create a directory (/tmp/packages/scale) and copy the listed packages.
+
+   ```cli
+   $ mkdir -p /tmp/packages/scale
+   ```
+
+   Example:
+   ```cli
+   $ ls /tmp/packages/scale
+   SpectrumScale_public_key.pgp
+   gpfs.adv-5.1.7-0.x86_64.rpm
+   gpfs.base-5.1.7-0.x86_64.rpm
+   gpfs.crypto-5.1.7-0.x86_64.rpm
+   gpfs.docs-5.1.7-0.noarch.rpm
+   gpfs.gpl-5.1.7-0.noarch.rpm
+   gpfs.gskit-8.0.55-19.1.x86_64.rpm
+   gpfs.gss.pmcollector-5.1.7-0.el8.x86_64.rpm
+   gpfs.gss.pmsensors-5.1.7-0.el8.x86_64.rpm
+   gpfs.gui-5.1.7-0.noarch.rpm
+   gpfs.java-5.1.7-0.x86_64.rpm
+   gpfs.license.dm-5.1.7-0.x86_64.rpm
+   gpfs.msg.en_US-5.1.7-0.noarch.rpm
+    ```
+   
+   **Note**: Ensure that you use the same Spectrum Scale packages for the worker custom image.
+
+
+6. Download a pre-built [Packer binary](https://www.packer.io/downloads) for your operating system.
+
+## Create Custom Image (using Packer)
+
+The following steps provision the IBM Cloud VSI, install IBM Spectrum Scale RPM's, and create a new image.
+
+1. Change working directory to `custom_image/storage/`.
+
+   ```cli
+   cd hpc-cluster-symphony/custom_image/storage/
+   ```
+
+2. Create packer variable definitions file (`inputs.auto.pkrvars.hcl`) and provide infrastructure inputs.
+    
+   **Note**: The `vpc_subnet_id` and `source_image_name` should belong to `vpc_region` variable value. 
+
+   Minimal Example:
+
+   ```jsonc
+   $ cat inputs.auto.pkrvars.hcl
+   ibm_api_key = "<IBMCloud_api_key>"
+   vpc_region = "<IBMCloud_supported_region_name>"
+   resource_group_id = "<Existing_resource_group_id>"
+   vpc_subnet_id = "<Existing_subnet_id_from_provided_vpc_region>"
+   source_image_name = "<Source_image_name_from_images_of_VPC>"
+   image_name = "<image_name_for_newly_created_custom_image>"
+    ```
+
+3. Run `packer init .` to install the Packer plugin for IBM Cloud.
+
+
+4. Run `packer build .` to create the custom image.
+
+   **Note**: The `mmbuildgpl` command's output in the Packer console (mmbuildgpl: Building GPL module completed successfully) indicates if the new custom image creation is successful. Also, the newly created image should be listed under the custom [Images of VPC](https://cloud.ibm.com/vpc-ext/compute/images) with the name that you provided for the `image_name` parameter with Active status.
+
+
+## Limitations:
+
+- The custom image creation scripts are only compatible with Red Hat-based Linux distributions operating systems.
+- The custom image creation scripts support only VSI based deployment models for both Symphony (where worker_type is specified as VSI) and Scale (where storage_type is specified as scratch).
```

Comment:
Do we have a Github issue open (and in backlog) to add custom image creation script support for deployment models that make use of bare metals (for worker nodes, and for Scale persistent storage nodes)?

Reply:
Yes Augie, We do have.